esphome/components/scd30/automation.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "esphome/core/component.h"
+#include "esphome/core/automation.h"
+#include "scd30.h"
+
+namespace esphome {
+namespace scd30 {
+
+template<typename... Ts> class SetForcedRecalibrationValueAction : public Action<Ts...> {
+ public:
+  explicit SetForcedRecalibrationValueAction(SCD30Component *scd30) : scd30_(scd30) {}
+
+  TEMPLATABLE_VALUE(uint16_t, forced_recalibration_value);
+
+  void play(Ts... x) override {
+    if (this->forced_recalibration_value_.has_value()) {
+      this->scd30_->set_forced_recalibration_value(this->forced_recalibration_value_.value(x...));
+    }
```

Comment:
If it is `required` in config validation, then no need to check if it has a value here
```suggestion
    this->scd30_->set_forced_recalibration_value(this->forced_recalibration_value_.value(x...));
```

Reply:
Done